### PR TITLE
Use dnf instead of yum for ubi8.

### PR DESCRIPTION
**Pull request by sebrandon1**
Use dnf instead of yum for ubi8.

### CODE REVIEW

File: test-partner/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 
-RUN yum install -y wget make hostname iproute iputils openssh openssh-clients podman psmisc
+RUN yum install -y wget make hostname iproute iputils openssh openssh-clients podman psmisc --skip-broken
```
[the remark this answers is not in the document]
Comment (sebrandon1) [reply test-partner/Dockerfile]: I believe we still use it as part of the old repo: https://github.com/test-network-function/test-network-function/blob/main/pkg/tnf/handlers/cnffsdiff/cnffsdiff.go#L99

**Comment by sebrandon1** [comment]
Found this article: https://lwn.net/Articles/894725/

It mentions `2023112 - Podman should be built with the support of 'libsubid'` was one of the items fixed?

**Comment by sebrandon1** [comment]
Looks like this bug has been reported here: https://bugzilla.redhat.com/show_bug.cgi?id=2084179

**Comment by sebrandon1** [comment]
I think I will keep this change as-is right now until the above BZ issue is fixed.  Seems like its a mirror/packaging issue upstream.  As far as this change I've noticed that they recommend yum for RHEL7 and dnf for RHEL8 so I'll stick with `dnf` for this change.

**Comment by bnshr** [comment]
Interesting recommendation. In RHEL8 , yum is just a symlink to dnf.

**Pull request merged**